packages/integrations/netlify/src/netlify-functions.ts

```diff
@@ -99,6 +102,8 @@ export const createExports = (manifest: SSRManifest, args: Args) => {
 			headers: responseHeaders,
 			body: responseBody,
 			isBase64Encoded: responseIsBase64Encoded,
+			// @ts-expect-error Property 'netlify' does not exist on type '{}'
+			ttl: locals.netlify?.builders?.ttl,
```

Comment:
Could we parse the `Cache-Control` here instead of exposing a new API? Looking at your ISR experiments and docs, that seems to be supported for Vercel and Cloudflare ootb, it would be nice to keep the same syntax for Netlify, but glueing this special `ttl` handling here.

Reply:
That's a great point! I agree it would be nice to have a uniform approach for all the adapters, but that approach should be the subject of an RFC.

Reply:
I am hesitant because parsing the `Cache-Control` header is not without its caveats.

Builders only support server-side control, the closest directive is `s-maxage`, and the others would not just be thrown out but replaced with `public, max-age=0, must-revalidate`. Users might find it confusing to have their directives ignored or even inverted.

Also, I want to leave room for an RFC that abstracts over cache headers (and other ISR methods) similar to what Next and Nuxt have. By abstracting parsing the header, we are implying that `Cache-Control` _is_ the abstraction.

Reply:
Alright. Thanks for the explanation. In that case we can go with the PR's approach first and we can revisit this later.

Reply:
Yeah, I agree